### ISSUE TITLE
Get information on what's needed for a tunnel mural

### ISSUE THREAD

**Issue by ct-martin** [RITlug/tasks#25]
Contact the Resident Hall Association about what we need for approval to do a tunnel mural.
Due date: Friday 2/2/2018 at 2:30pm (eboard meeting)

**Comment by axk4545** [comment]
Here is a link to the RHA website: [http://www.rit.edu/studentaffairs/rha/](http://www.rit.edu/studentaffairs/rha/)
On the right side is a link to a mural request form.

**Comment by jflory7** [comment]
I looked at the form request but it wasn't clear if an artist is included with the request. I emailed RHA to find out. If it's included, I think this is easier to move forward with. Otherwise, we're blocked on an artist.

**Comment by axk4545** [comment]
For the mural we paint it ourselves they give us a projector and paint for a set range of time. I believe we get two "appointments" to use it. we project our template on the wall and paint based on that. We can talk to them about possibly using an outside artist though.

**Comment by jflory7** [comment]
@axk4545 @ct-martin This task is currently on the ["In progress" board](https://github.com/orgs/RITlug/projects/1#card-7473930). Do we have any progress on this or should we shift this to the backlog?

**Comment by jflory7** [comment]
I'm going to shift this to the backlog as it's still blocked and there are no further updates.

**Comment by axk4545** [comment]
@jflory is it blocked on me?

**Comment by ct-martin** [comment]
I think it's blocked on me making a mock-up/draft. It's a low priority for
me b/c time and this is also a long term thing.

--
Christian Martin (Phone)

**Comment by Tjzabel** [comment]
# Image Idea
![RITlug Logo](https://raw.githubusercontent.com/RITlug/brand/master/logo/png/vertical_color_no_wordmark.png)

### This image is very simple, and would not require much expertise to paint freehand.

**Comment by jflory7** [comment]
@ct-martin Since we are not making a profit from this work, I am fairly certain we could use the image via [fair use doctrine](https://www.copyright.gov/fair-use/more-info.html) (see [Section 107 of the Copyright Act](https://www.copyright.gov/title17/92chap1.html#107)).

It may be a complicated design to do – the image I am thinking of would be difficult to paint onto a wall, but maybe you have something different in mind. I also wonder if the _Hackers_ reference is too obscure – I think it's an inside joke among a few of us in the club and many other club members may miss the reference. I like the idea of using our logo at a minimum.

**Comment by ct-martin** [comment]
Part of the work is simplifying the design. Also, the copyright thing is
more of a club office issue since that's now part of the mural design
requirements.

--
Christian Martin (Phone)

**Comment by ct-martin** [comment]
Also, the logo will be in the edited version

--
Christian Martin (Phone)

**Comment by axk4545** [comment]
I know I am outgoing eboard but FWIW the logo might make more sense than our original idea.

**Comment by Tjzabel** [comment]
At this point, I almost want to say we should close this issue. I'm not sure if this is something we want to do/will have bandwidth to do for quite a long time.

With that being said, is this something we still have an interest in pursuing?

**Comment by ct-martin** [comment]
I think this should be considered in the long-term planning meeting as a
way to get our name out. It likely will be slated for next semester at that
time, or maybe dropped entirely, but it would be nice if we could.

--
Christian Martin (Phone)

**Comment by Tjzabel** [comment]
I would like to close this issue. I do not think this is something in scope for RITlug at this time. 

If a desire comes to make this a reality, this issue can be reopened.
